src/assessments/assessment-builder.tsx

```diff
@@ -4,27 +4,27 @@ import * as _ from 'lodash';
 
-import { AssessmentToggleActionPayload } from '../background/actions/action-payloads';
-import { createInitialAssessmentTestData } from '../background/create-initial-assessment-test-data';
-import { InstanceIdentifierGenerator, UniquelyIdentifiableInstances } from '../background/instance-identifier-generator';
-import { RequirementComparer } from '../common/assessment/requirement-comparer';
-import { AssesssmentVisualizationConfiguration } from '../common/configs/assesssment-visualization-configuration';
-import { Messages } from '../common/messages';
-import { ManualTestStatus } from '../common/types/manual-test-status';
-import { FeatureFlagStoreData } from '../common/types/store-data/feature-flag-store-data';
-import { AssessmentScanData, ScanData } from '../common/types/store-data/visualization-store-data';
-import { AssessmentInstanceRowData, AssessmentInstanceTable } from '../DetailsView/components/assessment-instance-table';
-import { AssessmentTestView } from '../DetailsView/components/assessment-test-view';
-import { RequirementLink } from '../DetailsView/components/requirement-link';
-import { AnalyzerProvider } from '../injected/analyzers/analyzer-provider';
-import { DecoratedAxeNodeResult, ScannerUtils } from '../injected/scanner-utils';
+import { AssessmentToggleActionPayload } from 'background/actions/action-payloads';
+import { createInitialAssessmentTestData } from 'background/create-initial-assessment-test-data';
+import { InstanceIdentifierGenerator, UniquelyIdentifiableInstances } from 'background/instance-identifier-generator';
+import { RequirementComparer } from 'common/assessment/requirement-comparer';
+import { AssesssmentVisualizationConfiguration } from 'common/configs/assesssment-visualization-configuration';
+import { Messages } from 'common/messages';
+import { FeatureFlagStoreData } from 'common/types/store-data/feature-flag-store-data';
+import { AssessmentScanData, ScanData } from 'common/types/store-data/visualization-store-data';
+import { AssessmentInstanceRowData, AssessmentInstanceTable } from 'DetailsView/components/assessment-instance-table';
+import { AssessmentTestView } from 'DetailsView/components/assessment-test-view';
+import { RequirementLink } from 'DetailsView/components/requirement-link';
+import { AnalyzerProvider } from 'injected/analyzers/analyzer-provider';
+import { DecoratedAxeNodeResult, ScannerUtils } from 'injected/scanner-utils';
 import {
     PropertyBags,
     VisualizationInstanceProcessor,
     VisualizationInstanceProcessorCallback,
-} from '../injected/visualization-instance-processor';
-import { DrawerProvider } from '../injected/visualization/drawer-provider';
-import { DictionaryStringTo } from '../types/common-types';
+} from 'injected/visualization-instance-processor';
+import { DrawerProvider } from 'injected/visualization/drawer-provider';
+import { DictionaryStringTo } from 'types/common-types';
+import { ManualTestStatus } from '../common/types/manual-test-status';
```

Comment:
It looks like you switched to the `common/...` style for most of the imports from common, but are still using `../` style for `manual-test-status` imports throughout the files in this PR - is this one meant to use `common/` as well?

Reply:
Good catch! Updated.